Cargo.toml

```diff
@@ -17,13 +17,12 @@ name = "datafusion_catalogprovider_glue"
 path = "src/lib.rs"
 
 [dependencies]
-aws-config = "0.49.0"
-aws-sdk-glue = "0.19.0"
-aws-types = "0.49.0"
-datafusion = { version = "12.0.0", features = ["avro"] }
-object_store = { version = "0.5.0", features = ["aws"] }
+aws-config = { version = "1.1.7", features = ["behavior-version-latest"] }
+aws-sdk-glue = "1.25.0"
+aws-types = "1.2.0"
+datafusion = { version = "37.0.0", features = ["avro"] }
```

Comment:
I have bumped to 36 (as I would like to add support for deltalake and iceberg, both crates are on datafusion 36)